frontend/appflowy_flutter/lib/workspace/presentation/settings/pages/settings_workspace_view.dart

```diff
@@ -1266,3 +1266,90 @@ class _SelectionColorValueWidget extends StatelessWidget {
     );
   }
 }
+
+class _DocumentPaddingSetting extends StatelessWidget {
+  const _DocumentPaddingSetting();
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<DocumentAppearanceCubit, DocumentAppearance>(
+      builder: (context, state) {
+        return Column(
+          children: [
+            Row(
+              children: [
+                FlowyText.medium(
+                  LocaleKeys.settings_appearance_documentSettings_width.tr(),
+                ),
+                const Spacer(),
+                SettingsResetButton(
+                  onResetRequested: () =>
+                      context.read<DocumentAppearanceCubit>().syncWidth(null),
+                ),
+              ],
+            ),
+            const VSpace(6),
+            Container(
+              height: 32,
+              padding: const EdgeInsets.only(right: 4),
+              child: _DocumentPaddingSlider(
+                onPaddingChanged: (value) {
+                  context.read<DocumentAppearanceCubit>().syncWidth(value);
+                },
+              ),
+            ),
+          ],
+        );
+      },
+    );
+  }
+}
+
+class _DocumentPaddingSlider extends StatefulWidget {
+  const _DocumentPaddingSlider({
+    required this.onPaddingChanged,
+  });
+
+  final void Function(double) onPaddingChanged;
+
+  @override
+  State<_DocumentPaddingSlider> createState() => _DocumentPaddingSliderState();
+}
+
+class _DocumentPaddingSliderState extends State<_DocumentPaddingSlider> {
+  double width = 480;
```

Comment:
```suggestion
  late double width;
```

Reply:
I updated the max and min values and made them const values.